workspaces/orchestrator/app-config.yaml

```diff
@@ -124,14 +124,25 @@ dynamicPlugins:
   frontend: {}
 orchestrator:
   sonataFlowService:
+    runtime: docker
+    # uncomment for podman
+    # runtime: podman
     baseUrl: http://localhost
     port: 8899
     autoStart: true
+    # If you're using a MacBook with M-series chips, uncomment the line below to use ARM images.
+    # To pull the image, you'll first need to authenticate with the registry.
+    # For more information, visit: https://red.ht/410mMNU.
+    # container:  registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:1.35.0
     # notsecret
     notificationsBearerToken: bXljdXJscGFzc3dkCg==
     notificationsUrl: http://host.docker.internal:7007
+    # uncomment for podman:
+    # notificationsUrl: http://host.containers.internal:7007
+    persistence:
+      path: ./.devModeTemp/db_persistence # this will be under packages/backend
```

Comment:
Unsure about the `persistance`. According to [our docs](https://www.rhdhorchestrator.io/blog/devmode-vs-prodmode/), devMode has no persistence. @masayag wdyt?

Reply:
In dev mode, the postgres server is an embedded one by default.
But it is there, and workflow runs can survive pod restart.
So this property just exposes it - which seems good to me.

Reply:
Thanks for confirming @masayag i will make some changes to make it optional so that developers can chose what they want